apps/andi/lib/andi/input_schemas/datasets.ex

```diff
@@ -134,11 +134,21 @@ defmodule Andi.InputSchemas.Datasets do
       {:ok, get(dataset_id)}
   end
 
+  def is_unique?(_id, data_name, org_name) when is_nil(data_name) or is_nil(org_name), do: true
```

Comment:
Another TODO to "unit" test this as part of the `Datasets` module integration test